gremlin-go/driver/request.go

```diff
@@ -32,11 +29,36 @@ type request struct {
 	args      map[string]interface{} `json:"args"`
 }
 
-func makeStringRequest(requestString string) (req request) {
-	return request{uuid.New(), op, processor, map[string]interface{}{
-		"gremlin": requestString,
-		"aliases": map[string]interface{}{
-			"g": "g",
+const stringOp = "eval"
+const stringProcessor = ""
+
+func makeStringRequest(stringGremlin string) (req request) {
+	return request{
+		requestID: uuid.New(),
```

Comment:
Would it be better to use uuidv1 instead of uuidv4, since it can produce extra information (like whether the requests are coming from the same machine, and also includes timestamp)?

Reply:
The other drivers use v4